src/wallet/walletdb.cpp

```diff
@@ -33,46 +33,46 @@ static std::atomic<unsigned int> nWalletDBUpdateCounter;
 bool CWalletDB::WriteName(const std::string& strAddress, const std::string& strName)
 {
     nWalletDBUpdateCounter++;
-    return Write(make_pair(std::string("name"), strAddress), strName);
+    return batch.Write(make_pair(std::string("name"), strAddress), strName);
```

Comment:
I don't understand why this doesn't cause a compile error. We've removed all the "using namespace std".